src/serve_rendered.js

```diff
@@ -893,13 +893,13 @@ export const serve_rendered = {
             // console.log('Handling request:', req);
             if (protocol === 'sprites') {
               const dir = options.paths[protocol];
-              const file = unescape(req.url).substring(protocol.length + 3);
+              const file = decodeURI(req.url).substring(protocol.length + 3).replaceAll('%2c', ',');
```

Comment:
What is this replacement added for?

Reply:
Sorry, I did not check the code before. Correct would be to use `decodeURIComponent` as counterpart of `encodeURIComponent`.
`decodeURI` / `encodeURI` does not escape commas, so I added the extra replace after using a style with two fonts configured and receiving `ERROR: Font not found: Noto Sans Regular%2cNoto Sans Bold`.

I will fix that later today an force push the squashed commit.